examples/features/fileupload/ReadME.md

```diff
@@ -0,0 +1,22 @@
+# Fileupload
+trpc-go 支持流式 RPC，通过流式 RPC，客户端和服务器可以建立连续连接，连续发送和接收数据，让服务器提供连续的响应。
+
```

Comment:
![image](https://github.com/trpc-group/trpc-go/assets/20353612/d035a6c5-0956-4a57-ae3f-26e2383a4749)

README.md, not ReadME.md